src/hooks/__tests__/useScanApi.test.ts

```diff
@@ -328,6 +328,19 @@ describe('useGetScanJobsApi', () => {
 
     expect(mockOnAddAlert.mock.calls).toMatchSnapshot('callbackError');
   });
+
+  it('should call axios with page_size=1000 query parameter', () => {
+    const { apiCall } = hookResult;
+    const spyAxios = jest.spyOn(axios, 'get');
+
+    const scanId = 123;
+    apiCall(scanId);
+
+    expect(spyAxios).toHaveBeenCalledTimes(1);
+    expect(spyAxios).toHaveBeenCalledWith(`${process.env.REACT_APP_SCANS_SERVICE}${scanId}/jobs/`, {
```

Comment:
Non-blocking: Personally, I think `process.env` does not belong to assertions. This quickly goes in the direction that variable equals itself, instead of checking the externally observable effects.

As far as I can tell, we don't really have established pattern for these situations. We hardly do any assertions around outgoing data. The only other reference to `process.env` in tests is a mock in [`useLoginApi.test.ts`](https://github.com/quipucords/quipucords-ui/blob/6f50e1fabf7a1cdf9215884193fe44e1119fa38a/src/hooks/__tests__/useLoginApi.test.ts#L217).

Reply:
good call, I will use the proper value (I was too lazy to see what it was )

Reply:
Yeah, I think I had the same problem some time ago - when I run `eslint`, it does not complain, but when I run `eslint --fix`, it changes file content. I have strongly-worded opinions about this.

Reply:
Fixed. but turns out there is a snapshot test covering this. I thought this was uncovered because initially I did not saw that failure.
https://github.com/quipucords/quipucords-ui/commit/4af00502ef3925eb257fdf41610550bc56ec806a